protobuf/module_msg.proto

```diff
@@ -243,22 +243,11 @@ message MeasureCommandGetSummaryArg {
  * may not be a multiple of the resolution.
  */
 message MeasureCommandGetSummaryResponse {
```

Comment:
Could we avoid/minimize changing the protobuf message field names? There are many external project/repo using this, it's all broken. At least two of my project is not compatible with these changes.